util/cron/test-perf.cray-cs-hdr.gasnet-ibv.large.bash

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Run performance tests on a cray-cs
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs-hdr'
+
+source $CWD/common-perf.bash
+export CHPL_TEST_PERF_DIR=/cray/css/users/chapelu/NightlyPerformance/cray-cs/16-node-cs-hdr
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs-hdr.gasnet-ibv.large"
+
+source $CWD/common-cray-cs.bash y
+source $CWD/common-perf-cray-cs-hdr.bash
+
+export GASNET_PHYSMEM_MAX=124G
```

Comment:
Personally I would prefer seeing `CHPL_GASNET_SEGMENT=large` before this line even though that's the default, because (1) self-documenting, (2) redundancy as a form of future-proofing, and (3) symmetry in the scripts. But I'll approve without it.

Reply:
I go back and forth on this too with thinking about being explicit vs. making sure the default is what we think it is. For now I just copied the other cray-cs script and plan on leaving it as-is, and if we want to change in the future we should change both.